lighthouse-core/gather/gatherers/dobetterweb/external-anchors-with-no-rel-noopener.js

```diff
@@ -0,0 +1,39 @@
+/**
+ * @license
+ * Copyright 2016 Google Inc. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+'use strict';
+
+const Gatherer = require('../gatherer');
+
+class ExternalAnchorsWithNoRelNoopener extends Gatherer {
```

Comment:
Thx for adding the same-host check (technically just "same host" but not "same origin" as that [would include port like so](https://github.com/GoogleChrome/lighthouse/blob/master/lighthouse-core/audits/service-worker.js#L23-L31) -- i think it's okay to not consider port in this case. :) 

At this point the gatherer is just collecting ALL anchors with norelnoopener, so I think we should nuke the "External" in the name.